modules/javafx.graphics/src/main/java/javafx/scene/robot/Robot.java

```diff
@@ -251,7 +251,12 @@ public Color getPixelColor(double x, double y) {
 
```

Comment:
The fix looks good. There are two getPixelColor methods in Robot.java. Can you also update the docs for the other one?

Reply:
I knew I forgot something, lol.